app/src/main/java/com/duckduckgo/app/browser/omnibar/BottomAppBarBehavior.kt

```diff
@@ -76,6 +76,7 @@ class BottomAppBarBehavior<V : View>(context: Context, attrs: AttributeSet) : Co
     ) {
         super.onNestedPreScroll(coordinatorLayout, child, target, dx, dy, consumed, type)
 
+        // don't hide the app bar in the autocomplete suggestions list or the NTP
         if (target.id != R.id.autoCompleteSuggestionsList && target.id != com.duckduckgo.newtabpage.impl.R.id.newTabContentScroll) {
```

Comment:
The behavior is applied in the layout, as well, so probably not. But I’ve since changed this because the only target we really want is the webView. See the [latest changes](https://github.com/duckduckgo/Android/pull/5003/files#diff-fe6ebde9b11eddd212b40d9b12701a7e23e083cfc919c06b7a084d8f3e61154dR82).